external/fv3fit/tests/reservoir/helpers.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
```

Comment:
Moving the model object instance generation to a shared location for test files.